tailwind.config.js

```diff
@@ -76,6 +76,10 @@ module.exports = {
         '2px': '2px',
       },
     },
+    screens: {
+      xs: '480px',
```

Comment:
Would any of the elements that use this `xs` breakpoint work okay with the default `sm` TW breakpoint? 

We've converted the other pages using based off `sm` being the smallest breakpoint. It would help us keep all pages consistent if we can hopefully avoid introducing a smaller breakpoint.

When we were originally styling these pages in 2017(?) it was with Bootstrap 3 and [480px](https://getbootstrap.com/docs/3.4/customize/#media-queries-breakpoints) was the smallest breakpoint they had with that version. Interestingly, in the latest version they've increased their smallest breakpoint to [576px](https://getbootstrap.com/docs/5.0/layout/breakpoints/). Must be for the more prevelant bigger phone screens we have these days, thats fairly inline with the sm breakpoint TW gives us.